### ISSUE TITLE
Newest package version 0.10.5 not working

### ISSUE THREAD

**Issue by alexgawrilow** [dr-prodigy/python-holidays#422]
The newest version (0.10.5) of the holidays package is not working. Trying to import the package results in a `ModuleNotFoundError`

```
>>> import holidays
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/alex/.venv/lib/python3.8/site-packages/holidays/__init__.py", line 13, in <module>
    from holidays.countries import *
  File "/home/alex/.venv/lib/python3.8/site-packages/holidays/countries/__init__.py", line 22, in <module>
    from .burundi import Burundi, BI, BDI
  File "/home/alex/.venv/lib/python3.8/site-packages/holidays/countries/burundi.py", line 21, in <module>
    from holidays.utils import get_gre_date
  File "/home/alex/.venv/lib/python3.8/site-packages/holidays/utils.py", line 17, in <module>
    from hijri_converter import convert
ModuleNotFoundError: No module named 'hijri_converter'
```

**Comment by mweinelt** [comment]
That's because `setup.py` needs to require https://pypi.org/project/hijri-converter/.

**Comment by mweinelt** [comment]
The packaging is prone to this error, because it lists the dependencies separately in `.travis.yml`

https://github.com/dr-prodigy/python-holidays/blob/79dfc55777111d5a42b1ff3a0ed241fbea79b1ae/.travis.yml#L9-L17

The `python setup.py install` should take care of at least the normal runtime requirements.

**Comment by dr-prodigy** [comment]
Hi,
it should be fixed now and released as 0.10.5.1 on pypi.
Pls let me know, thank you very much!

**Comment by alexgawrilow** [comment]
Thank you. It's working in 0.10.5.1. I will close this.
